amundsen_application/api/metadata/v0.py

```diff
@@ -494,3 +472,68 @@ def _log_get_user(*, user_id: str) -> None:
         logging.exception(message)
         payload = jsonify({'msg': message})
         return make_response(payload, HTTPStatus.INTERNAL_SERVER_ERROR)
+
+
+@metadata_blueprint.route('/user/bookmark', methods=['GET'])
+def get_bookmark() -> Response:
+    try:
+        user_id = request.args.get('user_id')
+        if user_id is None:
+            if app.config['AUTH_USER_METHOD']:
+                user_id = app.config['AUTH_USER_METHOD'](app).user_id
+            else:
+                raise Exception('AUTH_USER_METHOD is not configured')
+
+        url = '{0}{1}/{2}/follow/'.format(app.config['METADATASERVICE_BASE'], USER_ENDPOINT, user_id)
+
+        response = request_metadata(url=url, method=request.method)
```

Comment:
If the response is not a success message, this would not go in the Exception block, and will raise exception when try to proceed normal try block. 
I think you should call `raise_for_status()` method of response, to properly raise the exception. 

Also, the whole try except block is repetitive, how about make this a decorator?

Reply:
Filed: https://github.com/lyft/amundsenfrontendlibrary/issues/177

Updating this pattern is probably out of scope, but that's a good idea. I'm not exactly certain we're doing the best with exception handling here.